src/tools-download.ts

```diff
@@ -127,7 +127,11 @@ export async function downloadAndExtract(
     core.warning(
       `Failed to download and extract CodeQL bundle using streaming. Falling back to downloading the bundle before extracting.`,
     );
-    core.warning(getErrorMessage(e));
+    core.warning(`Error: ${getErrorMessage(e)}`);
```

Comment:
Don't prefix a warning with `Error`.  

```suggestion
    core.warning(`Warning: ${getErrorMessage(e)}`);
```

But, do we even need the prefix at all?

Maybe reverting this is better:

```suggestion
    core.warning(getErrorMessage(e));
```

Reply:
Thank you for catching this.

Seeing it reported now, I agree it looks a bit silly.

My rationale for changing this was that this was rendering an error that was generated in the process of unpacking as a simple warning, with no other context whatsoever, which could be confusing to a user when interpreting the runner log ([example](https://github.com/github/codeql-action/actions/runs/12166081189/job/33931523101#step:7:35)).

I'm still motivated to decorate it somehow, as reverting may not satisfy this concern, but I will think about a cleaner decoration that doesn't look weird/silly.

Reply:
Okay, this is what I came up with: [88bcf64](https://github.com/github/codeql-action/pull/2624/commits/88bcf64e02c4dffb7bded80efda37efcdb69b07a)

I've reworded the warnings to flow a bit more naturally for me, and to allow me to embed the error message in an indicative but non-invasive way.

Is this workable for you @aeisenberg ? Or do you have a strong preference for the previous state of affairs?